core/src/Cabana_VerletList.hpp

```diff
@@ -685,11 +709,12 @@ class VerletList
         // count and fill at the same time, unless the array size is exceeded,
         // at which point only counting is continued to reallocate and refill.
         typename builder_type::FillNeighborsPolicy fill_policy(
-            builder.bin_data_1d.numBin(), Kokkos::AUTO, 4 );
+            builder.bin_data_1d.numBin(), Kokkos::AUTO, builder.chunk_size );
```

Comment:
So the test was failing in serial because it was trying to launch 4 threads when none were available?

Reply:
It failed because it was running through the inner loop 4 times, quadrupling the neighbor counts/lists